examples/accelerate_configs/deepspeed_zero1.yaml

```diff
@@ -8,7 +8,7 @@ distributed_type: DEEPSPEED
 downcast_bf16: 'no'
 machine_rank: 0
 main_training_function: main
-mixed_precision: 'no'
+mixed_precision: 'bf16'
```

Comment:
We can now set this as the default since we initialise both the reference and active models with DeepSpeed